utils_cv/detection/bbox.py

```diff
@@ -194,3 +194,27 @@ def from_array(
 
     def __repr__(self):
         return f"{super().__repr__()} | score: {self.score}"
+
+
+def boxes_iou(box1: DetectionBbox, box2: DetectionBbox):
```

Comment:
I am calling all variables "boxes" rather than "bboxes". Is that OK? Should we use "boxes" going forward everywhere instead of the longer "bboxes"?

Reply:
Or if in code we use "bboxes" (nearly) everywhere should I rename this also to "bboxes"?

Reply:
i think bboxes for consistency is preferable. its also more obviously "bounding box" in my opinion.